libnavigation-core/src/test/java/com/mapbox/navigation/core/routerefresh/RouteRefreshControllerTest.kt

```diff
@@ -70,11 +70,11 @@ class RouteRefreshControllerTest {
     @Test
     fun `route refreshes`() = runBlockingTest {
         val (initialRoute, refreshedRoute) = createTestInitialAndRefreshedTestRoutes()
-        val directionsSession = mockk<DirectionsSession>().apply {
-            onRefresh { _, _, _, callback -> callback.onRefreshReady(refreshedRoute) }
+        val routeRefreshStub = RouteRefreshStub().apply {
```

Comment:
oh no, not this "mocks vs stubs" holy war again 😱 Is has never finished good 😆 
Call me if you're interested in the topic of testing so that we could share our opinions with respect to each other even if we don't agree with each other.